src/patternfly/components/Popover/docs/code.md

```diff
@@ -1,19 +1,31 @@
 ## Overview
 
-Only if needed write a short description with implementation notes. Design and interaction notes are already written on the design specs, don't repeat information.
-
-For example for buttons: Always add a modifier class to add color to the button. Never use the class `.btn` on its own.
+A popover is used to provide contextual information for another component on click.  The popover itself is made up of two main elements: arrow and content. The content element follows the pattern of the popover box component, with a close icon in the top right corner, as well as a header and body.  One of the directional modifiers (`.pf-m-left`, `.pf-m-top`, etc.) is required on the popover component
 
 ## Accessibility
 
-| Attribute | Applied To | Outcome |
+| Attribute | Applies to | Outcome |
 | -- | -- | -- |
-| `role` or `aria` | `pf-c-popover` |  accessibility notes. |
+| `role="dialog"` | `.pf-c-popover` | Identifies the element that serves as the popover container. **Required**|
+| `aria-labeledby="[id value of .pf-c-popover__header-title]"` | `.pf-c-popover` | Gives the popover an accessible name by referring to the element that provides the dialog title. **Required when .pf-c-popover__header-title is present** |
+| `aria-label="[title of popover]"` | `.pf-c-popover` | Gives the popover an accessible name. **Required when .pf-c-popover__header-title is _not_ present** |
+| `aria-describedby="[id value of applicable content]"` | `.pf-c-popover` | Gives the popover an accessible description by referring to the popover content that describes the primary message or purpose of the dialog. Not used if there is no static text that describes the popover. |
+| `aria-popover="true"` | `.pf-c-popover` | Tells assistive technologies that the windows underneath the current popover are not available for interaction. **Required**|
```

Comment:
This looks like a Find/Replace mistake. 😉  I don't see `aria-popover` listed as an [aria attribute](https://www.w3.org/TR/wai-aria-1.1/#state_prop_def).

I think you meant `aria-modal="true"` instead.

Reply:
I opened https://github.com/patternfly/patternfly-next/issues/527 to track this one separately from this PR.